jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/python/PythonLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -151,47 +133,12 @@ class PythonLocalLambdaRunConfigurationIntegrationTest(private val runtime: Runt
 
         assertThat(executeLambda.exitCode).isEqualTo(0)
         assertThat(jsonToMap(executeLambda.stdout))
```

Comment:
I don't think this is that controversial, but maybe we should use multiple assertThats with descriptions set so we can see better from test reports?

Reply:
Sure, I wanted to see how much this actually sped up the builds first. Will edit it if it works well